### ISSUE TITLE
Friendly error: Dependency is an expression

### ISSUE THREAD

**Issue by Triloworld** [yashha/wpapi-extensions#7]
Another think of that wasn't found reported:

> WARN  in ./node_modules/wpapi-extensions/dist/index.js - friendly-errors 10:01:10
> Critical dependency: the request of a dependency is an expression  

I use : https://github.com/yashha/wp-nuxt
It work, just that friendly error in start of nuxt in development mode.

How this can be fixed?
Index.js as link and this show up:

```
parcelRequire=function(e,r,t,n){var i,o="function"==typeof parcelRequire&&parcelRequire,u="function"==typeof require&&require;function f(t,n){if(!r[t]){if(!e[t]){var i="function"==typeof parcelRequire&&parcelRequire;if(!n&&i)return i(t,!0);if(o)return o(t,!0);if(u&&"string"==typeof t)return u(t);var c=new Error("Cannot find module '"+t+"'");throw c.code="MODULE_NOT_FOUND",c}p.resolve=function(r){return e[t][1][r]||r},p.cache={};var l=r[t]=new f.Module(t);e[t][0].call(l.exports,p,l,l.exports,this)}return r[t].exports;function p(e){return f(p.resolve(e))}}f.isParcelRequire=!0,f.Module=function(e){this.id=e,this.bundle=f,this.exports={}},f.modules=e,f.cache=r,f.parent=o,f.register=function(r,t){e[r]=[function(e,r){r.exports=t},{}]};for(var c=0;c<t.length;c++)try{f(t[c])}catch(e){i||(i=e)}if(t.length){var l=f(t[t.length-1]);"object"==typeof exports&&"undefined"!=typeof module?module.exports=l:"function"==typeof define&&define.amd?define(function(){return l}):n&&(this[n]=l)}if(parcelRequire=f,i)throw i;return f}({"7QCb":[function(require,module,exports) {
"use strict";function e(e){e||"function"==typeof e.registerRoute||console.log("Please provide a wpapi instance.")}function t(t){e(t),t.cpt=function(e){return t[e]=t.registerRoute("wp/v2","/"+e+"/(?P<id>)",{params:["embed","before","after","author","parent","post"]}),t[e]()}}function r(t){e(t),t.frontPage=t.registerRoute("wuxt/v1","/front-page",{params:["embed"]})}function o(t){e(t),t.menu=t.registerRoute("wuxt/v1","/menu",{params:["location"]})}function n(t){e(t),t.slug=t.registerRoute("wuxt/v1","/slug/(?P<name>)",{params:["embed"]})}function s(s){e(s),r(s),o(s),n(s),t(s)}Object.defineProperty(exports,"__esModule",{value:!0}),exports.registerCpt=t,exports.registerFrontPage=r,exports.registerMenu=o,exports.registerSlug=n,exports.registerWuxt=s;
},{}]},{},["7QCb"], null)
//# sourceMappingURL=/index.js.map
```

Add "dependabot" to this package with will add pull request automatically for security updates

**Comment by Triloworld** [comment]
Bug come from: https://github.com/parcel-bundler/parcel/issues/2883
If fix will be in minor version then issue can be close. Otherwise need bump dependency

**Comment by Triloworld** [comment]
Resolve in yashha/wp-nuxt#114

**Comment by Triloworld** [comment]
Ironically, this fix not only remove error but add some features!

https://parceljs.org/cli.html#enable-experimental-scope-hoisting/tree-shaking-support

I will add pull request with fix right away. Sorry for taking so long for it

**Comment by yashha** [comment]
I guess this is now fixed by #12, thank you very much
